daml/Marketplace/Utils.daml

```diff
@@ -40,3 +41,12 @@ getAccountOwnerProvider: Text -> (Party, Party)
 getAccountOwnerProvider label =
   let [owner, provider] = map (fromSome . partyFromText . replace "'" "") (splitOn "@" label)
   in (owner, provider)
+
+getDepositQuantities : [ContractId AssetDeposit] -> Update (Numeric 10)
```

Comment:
We should use `Decimal` here - while it's just a type synonym of `Numeric 10`, `AssetDeposit` uses `Decimal` for `quantity` so it's best to match that, in case either of them get changed in the future.